src/index.html

```diff
@@ -11,7 +11,138 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form>
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <label class="personal-information label">
+          Surname:
+          <input 
+            type="text" 
+            name="surname" 
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="personal-information label">
+          Name:
+          <input 
+            type="text" 
+            name="name" 
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="personal-information label">
+          How old are You?
+          <input 
+            type="number" 
+            name="age" 
+            value="12" 
+            min="1" 
+            max="100"
+          >
+        </label>
+
+        <label class="personal-information label">
+          Full date of birth:
+          <input type="date" name="date">
+        </label>
+
+        <label class="personal-information label">
+          I accept the term of the agreement
+          <input type="checkbox" name="terms">
+        </label>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <label class="registration label">
+          E-mail:
+          <input 
+            type="email" 
+            name="surname" 
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="registration label">
+          Password:
+          <input 
+            type="password" 
+            name="name" 
+            required minlength="5" 
+            maxlength="50"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="facts label">
+          Do you love cats?
+          <label>
+            <input type="radio" 
+            name="pet" value="Yes"
+            >Yes
```

Comment:
```suggestion
            <input 
              type="radio" 
              name="pet" 
              value="Yes"
            >
              Yes
```
please fix code style everywhere

Reply:
Done